taichi_3d_gaussian_splatting/GaussianPointCloudRasterisation.py

```diff
@@ -652,6 +660,8 @@ def gaussian_point_rasterisation_backward(
                     # \frac{dC}{da_i} = c_i T(i) - \frac{1}{1 - a_i} w_i
                     alpha_grad_from_rgb = (color * T_i - w_i / (1. - alpha)) \
                         * pixel_rgb_grad
+                    alpha_grad_from_rgb -= pixel_rgb_grad * background_color_vector * \
```

Comment:
Could you provide the formula to calculate the gradient properly? I am a bit confused by the name convention.